test/format/README

```diff
@@ -0,0 +1,84 @@
+Tracing format operations can be useful in debugging.
+
+Turn on tracing with the "-t" flag:
```

Comment:
Shouldn't this be the `-T` configuration instead?  I think the `-t` option expects a number.

Reply:
`-T` is the parameter for `format.sh` to turn on format tracing.

Reply:
@AlexeyAnisimovAU and @mm-ng are both correct; the format binary in the develop branch uses both `-t` and `-T` for tracing, and `format.sh` uses  `-T` for tracing, and `-t` for the minutes to run timeout. I didn't look, but it's probably historical: `format.sh` probably had `-t` for a timeout from when it was first created, and I hacked in `-T` later to get tracing.

In the format-mirror branch, I removed `-T` as an option for format itself at some point, but left `-T` alone in `format.sh`, since `-t` wasn't available.

It would be an easy change to switch `format` to use `-T` instead of `-t`, then they'd be the same.

Reply:
Created a ticket for this: [WT-9561](https://jira.mongodb.org/browse/WT-9561)